src/app/shared/data/form-inputs.json

```diff
@@ -0,0 +1,82 @@
+{
+  "personal-details": [
+    {
+      "type": "text",
+      "key": "firstName",
+      "label": "First name:",
+
+      "order": 1
```

Comment:
what are you using the `"order"` property for?

Reply:
I had placed it for a logic where you could set the order of the fields, regardless of the position in the array.